Delicious/Utils/Constants.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Constants.swift
+//  Delicious
+//
+//  Created by HoaPQ on 7/27/20.
+//  Copyright © 2020 HoaPQ. All rights reserved.
+//
+
+import UIKit
+
+enum Icon {
+    static let icHomeNormal = #imageLiteral(resourceName: "ic_home_grey")
+    static let icHomeSelected = #imageLiteral(resourceName: "ic_home_orange")
+    
+    static let icSearch = #imageLiteral(resourceName: "ic_search")
+    static let icBack = #imageLiteral(resourceName: "ic_arrow_back")
+    
+    static let recipePlaceHolder = #imageLiteral(resourceName: "img_recipe_placeholder")
+    
+    static let icFavoriteSelected = #imageLiteral(resourceName: "ic_favorites_grey")
+    static let icFavorite = #imageLiteral(resourceName: "ic_favorite")
+    static let icFavoriteSelectedOrange = #imageLiteral(resourceName: "ic_favorites_orange")
+    
+    static let icShoping = #imageLiteral(resourceName: "ic_recipes_grey2")
+    static let icShopingSelected = #imageLiteral(resourceName: "ic_recipes_orange")
+    
+    static let icError = #imageLiteral(resourceName: "img_error")
+    static let icNetwork = #imageLiteral(resourceName: "img_no_connection")
+    static let icEmpty = #imageLiteral(resourceName: "img_no_items_cart")
+}
+
+enum Constant {
+    static let titleFontSize: CGFloat = 20
+    static let appTitle = "Delicious"
+    static let homeTitle = "Home"
+    static let favoriteTitle = "Favorite"
+    static let shoppingListTitle = "Shopping List"
+}
```

Comment:
nên tách ra 2 file vì 2 mục này sẽ nở ra rất to theo thời gian.

Reply:
hơn nữa phần title screen nếu chỉ dùng trong nội bộ một màn hình thì có thể để nó vào inner Constants của màn hình đó.

Reply:
Mấy cái này dùng cho cả tabbar nữa anh ạ!